analysis/organize_data.R

```diff
@@ -32,17 +32,16 @@ ipumsdata$marriage[ipumsdata$marst==4] <- "Divorced"
 ipumsdata$marriage[ipumsdata$marst==5] <- "Widowed"
 ipumsdata$marriage[ipumsdata$marst==6] <- "Single"
 ipumsdata$marriage <- factor(ipumsdata$marriage,
-                             levels=c("MarriedSP","MarriedSA","Seperated","Divorced","Widowed","Single"))
+                             levels=c("MarriedSP","MarriedSA","Separated","Divorced","Widowed","Single"))
```

Comment:
Simple spelling error but made one of your categories disappear. Your checking script caught this but apparently you didn't notice.